src/client/eppo-client.spec.ts

```diff
@@ -246,11 +246,7 @@ describe('EppoClient E2E test', () => {
 
     it('obfuscates assignments', () => {
       // Use a known salt to produce deterministic hashes
-      setSaltOverrideForTests({
```

Comment:
static test salt was incorrect so we rely on the `Salt` constructor to properly encode the bytes passed to it (for which there is also now a unit test).